entries/a/aidvantage.studentaid.gov.json

```
@@ -0,0 +1,15 @@
+{
+  "Aidvantage": {
+    "domain": "aidvantage.studentaid.gov",
+    "img": "aidvantage.studentaid.gov.png",
+    "contact": {
+      "form": "https://myaccount.aidvantage.studentaid.gov/EmailUs/SendEmail"
```

```suggestion
      "form": "https://aidvantage.studentaid.gov/contact-us"
```

The page you linked requires authentication.